PokeAlarm/WebhookStructs.py

```diff
@@ -176,9 +176,15 @@ def egg(data):
         else:
             id_ = data.get('gym_id')  # RM sends the gym id
 
+        if 'gym_name' in data:  # monocle sends gym_name in egg data.
+            gymname = check_for_none(str, data.get('gym_name'), 'unknown')
+        else:
+            gymname = 'unknown'
+
         egg = {
             'type': 'egg',
             'id': id_,
+            'gym_name': gymname,
```

Comment:
The point of the `check_for_none` is so that an explicit check isn't required - it should let you get the contents and default to `unknown` if it doesn't exists.

So this could be one line on : `'gym_name': check_for_none(str, data.get('gym_name'), 'unknown')`